crates/solidity/testing/snapshots/cst_output/ReceiveFunctionDefinition/simple/generated/0.4.22.yml

```diff
@@ -0,0 +1,19 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ receive () {                                                                     │ 0..12
+  2  │     // TODO: should have failed in earlier versions                              │ 13..64
+  3  │     // https://github.com/NomicFoundation/slang/issues/359                       │ 65..123
+  4  │ }                                                                                │ 124..125
+
+Errors: # 1 total
+  - >
+    Error: Expected ProductionKind ReceiveFunctionDefinition is not valid in Solidity language version '0.4.22'.
```

Comment:
I wonder if there is a need to include the used language version in the error string message?

1. It is not providing value to the user, since they already know the version they initialized the parser with.
2. It would make it easier for us to diff errors between versions, thus not producing the duplicated files in tests like this. At least until we start reporting errors as proper ADTs instead of raw strings.

Reply:
done